### DISCUSSION TITLE
Unicodes of Wave Icons

### DISCUSSION THREAD

**Discussion by shamilh2o** [h2oai/wave#1506] [Q&A] [answered]
Do we have a set of unicodes for wave icons which we can use with "ui.text()"?

Need: ui.text("<icon_unicode>  Topic")

**Comment by mturoci** [comment, marked as answer]
Wave icons are SVG-based, not font-based so no unicodes. You can either use [emojis](https://getemoji.com/) that are natively supported in browsers or rip off raw SVGs from [here](https://github.com/microsoft/fluentui/tree/master/packages/react-icons-mdl2/src/components).

**Reply by shamilh2o** [reply]
Noted with Thanks @mturoci !
